my-code/src/components/Header/index.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
```

Comment:
Just a code style opinion: add a new line break between absolute and relatives imports. 
It will help you to see which modules are local or come from an external package.